app-services/simple-filter-xml/Dockerfile

```diff
@@ -1,46 +1,48 @@
-#
-# Copyright (c) 2019 Intel Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-FROM golang:1.13-alpine AS builder
-
-LABEL license='SPDX-License-Identifier: Apache-2.0' \
-  copyright='Copyright (c) 2019: Intel'
-
-# add git for go modules
-RUN apk update && apk add --no-cache make git gcc libc-dev libsodium-dev zeromq-dev
-WORKDIR /app-service-examples
-
-COPY go.mod .
-
-RUN go mod download
-
-COPY . .
-RUN apk info -a zeromq-dev
-
-RUN make app-services/simple-filter-xml
-
-# Next image - Copy built Go binary into new workspace
-FROM alpine
-
-LABEL license='SPDX-License-Identifier: Apache-2.0' \
-  copyright='Copyright (c) 2019: Intel'
-
-RUN apk --no-cache add zeromq
-
-COPY --from=builder /app-service-examples/app-services/simple-filter-xml/res /res
-COPY --from=builder /app-service-examples/app-services/simple-filter-xml/app-service /simple-filter-xml
-
+#

```

Comment:
Why is this changed and can't determine what was changed...

Reply:
line 43

Reply:
K, see it now. This is not needed since it is already done in the code.

Reply:
i did to show how it can be done in docker too - i dont think it hurts.

Reply:
K, got it. ;-)